lib/gmp/package.ini

```diff
@@ -0,0 +1,11 @@
+[package]
+name=gmp
+tags=maths,lib
+maintainer=PatrickBlanchette <PatrickBlanchette@users.noreply.github.com>
```

Comment:
The culprit is @PatrickBlanchette that have a bad git.name configuration (and a notsogood email address).